lib/tbot/botfs/botfs.go

```diff
@@ -126,8 +137,8 @@ func (s *ACLSelector) CheckAndSetDefaults() error {
 
 // openStandard attempts to open the given path for reading and writing with
```

Comment:
Is the godoc still correct here? Feels like it doesn't always open for reading and writing anymore.

Reply:
Ah, good call, fixed!